pkgs/applications/altcoins/default.nix

```diff
@@ -34,4 +34,9 @@ rec {
 
   stellar-core = callPackage ./stellar-core.nix { };
 
+  zcash = callPackage ./zcash {
+    withGui = false;
+    openssl = pkgs.openssl_1_1_0;
+    boost = pkgs.boost163;
```

Comment:
This doesn't seem to be incorporating the feedback we gave to the original PR. I remember suggesting that if this expression needed a particular version of boost, it should ask for it directly in the parameters